pymc3/tests/test_sampling.py

```diff
@@ -222,16 +224,16 @@ def test_return_inferencedata(self, monkeypatch):
             assert result.posterior.sizes["chain"] == 2
             assert len(result._groups_warmup) == 0
 
-            # check warning for version 3.10 onwards
-            monkeypatch.setattr("pymc3.__version__", "3.10")
```

Comment:
This was the only occurrence of `monkeypatch` in this test, so it can be removed from the function inputs.